fdbserver/workloads/BulkLoading.actor.cpp

```diff
@@ -494,15 +506,18 @@ struct BulkLoading : TestWorkload {
 				BulkLoadTaskTestUnit taskUnit =
 				    self->generateBulkLoadTaskUnit(self, folderPath, dataSize, KeyRangeRef(beginKey, endKey));
 				bulkLoadTaskStates.push_back(taskUnit.bulkLoadTask);
-				bulkLoadDataList.push_back(taskUnit.data);
-				completeRanges.push_back(taskUnit.bulkLoadTask.getRange());
+				taskRanges.push_back(taskUnit.bulkLoadTask.getRange());
+				taskUnits.push_back(taskUnit);
 			}
-			// Issue above 3 tasks in the same transaction
+			// Issue above 2 tasks in the same transaction
 			wait(self->submitBulkLoadTasks(self, cx, bulkLoadTaskStates));
 			TraceEvent("BulkLoadingWorkLoadSimpleTestIssuedTasks");
 			wait(store(oldBulkLoadMode, setBulkLoadMode(cx, 1)));
 			TraceEvent("BulkLoadingWorkLoadSimpleTestSetMode").detail("OldMode", oldBulkLoadMode).detail("NewMode", 1);
-			wait(self->waitUntilAllTaskComplete(self, cx));
+			std::vector<BulkLoadTaskState> errorTasks = wait(self->waitUntilAllTaskCompleteOrError(self, cx));
```

Comment:
We don't check the second error flag before we iterate the errorTasks? It'll be empty if no error? That should be fine?

Reply:
`waitUntilAllTaskCompleteOrError` returns all error tasks. `checkAllTaskCompleteOrError`, returns both error flag and errorTasks.